packages/valory/services/apy_estimation_goerli/service.yaml

```diff
@@ -0,0 +1,34 @@
+name: apy_estimation_goerli
+author: valory
+version: 0.1.0
+description: A set of agents reaching consensus on APY estimates
+aea_version: '>=1.0.0, <2.0.0'
+license: Apache-2.0
+fingerprint:
+  README.md: bafybeieuhno4vkqtv7qrhxd4ya53qijzkaplrbvgi6fgoru7y3s7op2izy
+fingerprint_ignore_patterns: []
+agent: valory/apy_estimation:0.1.0:bafybeie7v5ajtdmglgswpme7u3n5uvfbqor2lvcqw3wg2ahmls7bw3ydy4
+number_of_agents: 4
+---
+public_id: valory/apy_estimation_abci:0.1.0
+type: skill
+models:
+  params:
+    args:
+      pair_ids:
+        uniswap_subgraph:
+        - '0xae461ca67b15dc8dc81ce7615e0320da1a9ab8d5'
+        spooky_subgraph:
+        - '0x2a651563c9d3af67ae0388a5c8f89b867038089e'
+        - '0x2b4c76d0dc16be1c31d4c1dc53bf9b45987fc75c'
+      ipfs_domain_name: /dns/registry.autonolas.tech/tcp/443/https
+---
+public_id: valory/ledger:0.1.0
```

Comment:
Why any network? Are we sending any tx? Afaik no

Reply:
Removed on 66a90f1c69a7baee4c881d4e3aba685da3bd2578.